arkouda/segarray.py

```diff
@@ -232,6 +233,37 @@ def concat(cls, x, axis=0, ordered=True):
         else:
             raise ValueError("Supported values for axis are 0 (vertical concat) or 1 (horizontal concat)")
 
+    @staticmethod
+    def from_return_msg(repMsg) -> SegArray:
+        """
+        Return a SegArray instance pointing to components created by the arkouda server.
+        The user should not call this function directly.
+
+        Parameters
+        ----------
+        repMsg : str
+            ; delimited string containing the segments, values, and lengths details
+
+        Returns
+        -------
+        SegArray
+            A SegArray representing a set of pdarray components on the server
+
+        Raises
+        ------
+        RuntimeError
+            Raised if a server-side error is thrown in the process of creating
+            the categorical instance
+        """
+        # parts[0] is "segarray". Used by the generic attach method to identify the
+        # response message as a SegArray
+        parts = repMsg.split(";")
```

Comment:
Again, this should be `+` delimited for consistency.

Reply:
Updated to use `+` delimited string in the response message